module-4/backend/week_two.md

```diff
@@ -2,10 +2,62 @@
 
 Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
-TBD  
+What's one difference between ES5 and ES6?
 
-#### Review  
+ES5 uses function syntax while ES6 uses arrow functions.
```

Comment:
Nice! Remember that `function` syntax still works in ES6, and is sometimes necessary (e.g., `prototype` functions).